testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1582,22 +1582,6 @@ public void testMissingMetrics()
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
-    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
-    public void testOptimizeFailsOnV2IcebergTable()
```

Comment:
From Spark compat testing perspective, we should have a test of v2 table + optimize + spark reading the table.